### PR TITLE
Login icons

### PR DESCRIPTION
**Pull request by RazerMoon**
Login buttons now have icons.

### CODE REVIEW

File: kofta/src/vscode-webview/svgs/GitHubIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+
+/* SVG from iconmonstr, https://iconmonstr.com/license/ */
```
Comment (overlisted) [inline kofta/src/vscode-webview/svgs/GitHubIcon.tsx:3]: you could get them from their official "branding" page though

Comment (ajmnz) [reply kofta/src/vscode-webview/svgs/GitHubIcon.tsx:3]: Or from the `react-feather` library, which is already in use.

**Comment by RazerMoon** [comment]
Changed the icons to `react-feather` like @ajmnz suggested.

Now it looks like this:

**Comment by ajmnz** [comment]
> Changed the icons to `react-feather` like @ajmnz suggested.

They match the current icon style used across the app, but thought `react-feather` had a solid version.
Not sure if I like how they look. What do you think @RazerMoon ?

**Comment by RazerMoon** [comment]
> Not sure if I like how they look. What do you think @RazerMoon ?

I like the solid ones more, but the website uses mostly outline icons.

image
image

I think consistency is more important.

**Comment by RazerMoon** [comment]
Here is how that would look though:

**Comment by RazerMoon** [comment]
Added brand colors while I'm still at it:

**Comment by ajmnz** [comment]
> Added brand colors while I'm still at it:

Looking good!
Should Twitter be more prominent than GitHub? Maybe they should have the same background. Look how on my concept all the buttons look consistent and contribute to maintain a visual balance and symmetry.

Screenshot 2021-02-22 at 13 42 31

Same applies for the current case. I think they should be either all blue or all dark gray.

Screenshot 2021-02-22 at 13 42 31
Screenshot 2021-02-22 at 13 43 31


Let me know what you think! @RazerMoon

**Comment by RazerMoon** [comment]
@ajmnz That is a really cool design!

I agree that the Twitter button sticks out a little bit, I was going for more of a Vercel look.

image

I think if you're going to change the background color of those, you would have to do that to all the buttons.

Also, I'm gonna change back the icons to the initial ones 😅. I do think consistency is important, but I think brand buttons are a good exception. Vercel also uses that exception.

image

**Comment by ajmnz** [comment]
> I agree that the Twitter button sticks out a little bit, I was going for more of a Vercel look.

Vercel really nailed it! If you prefer going the branded color way, make sure to dim them a bit (just like Vercel did!). This way they won't stick out that much and will blend better with the current color scheme.

> Also, I'm gonna change back the icons to the initial ones 😅

Yeah, wasn't feeling those too. 👍🏼

**Comment by RazerMoon** [comment]
> Vercel really nailed it! If you prefer going the branded color way, make sure to dim them a bit.

Won't dimming it change the brand color though? 🤔

**Comment by ajmnz** [comment]
> Won't dimming it change the brand color though? 🤔

Yep, but big brands are strongly associated with a certain color to the point where dimming it won't make it difficult to recognize it. As an example, Vercel used `#8367D3` for GitLab, but its real purple brand color is `#9B51E0`. Same thing happens with BitBucket: Vercel uses `#1668E2` and real color is `#0047B3`.

Kind of an interesting topic haha

**Comment by RazerMoon** [comment]
> Vercel used #8367D3 for GitLab, but its real purple brand color is #9B51E0. 

Cool! I didn't know that.

What about one of these then? I personally like the middle one.

image

From top to bottom:
`#1CA0F2`
`#0C84CF`
`#09669f`

**Comment by ajmnz** [comment]
> What about one of these then? I personally like the middle one.

Middle one looks good to me!

**Comment by RazerMoon** [comment]
Current look:

**Comment by tanay-pingalkar** [comment]
> > Added brand colors while I'm still at it:
> 
> Looking good!
> Should Twitter be more prominent than GitHub? Maybe they should have the same background. Look how on my concept all the buttons look consistent and contribute to maintain a visual balance and symmetry.
> 
> Screenshot 2021-02-22 at 13 42 31
> 
> Same applies for the current case. I think they should be either all blue or all dark gray.
> 
> Screenshot 2021-02-22 at 13 42 31
> 
> Screenshot 2021-02-22 at 13 43 31
> 
> Let me know what you think! @RazerMoon

when this is coming to truth

**Pull request merged**